go.mod

```diff
@@ -2,12 +2,10 @@ module github.com/dell/cert-csi
 
 go 1.22.0
 
-toolchain go1.22.5
```

Comment:
Any impact from removing this toolchain?

Reply:
I hope you have tried to build the binary and use it for some basic sanity before raising the PR.

Reply:
No, if we don't mention toolchain, it will take the default go version present in our machine.

Reply:
I have build the binary successfully.

Reply:
I can see https://github.com/dell/cert-csi/blob/0233a8536958961763bc4d2e1b5dd5e320d6f7f0/Makefile#L55 so for docker image it should be fine.
Thanks for the confirmation.